boxsdk/object/chunked_upload.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import unicode_literals, absolute_import
+
+import hashlib
+
+
+class ChunkedUpload(object):
+    _item_type = 'chunked_upload'
+    _uploaded_part = None
+    _part_array = []
+    _sha1 = hashlib.sha1()
+
+    def __init__(self, upload_session, content_stream, file_size):
+        self._upload_session = upload_session
+        self._content_stream = content_stream
+        self._file_size = file_size
+
+    def start(self):
+        self._upload()
+        content_sha1 = self._sha1.digest()
+        return self._upload_session.commit(content_sha1=content_sha1, parts=self._part_array)
+
+    def _upload(self):
+        while len(self._part_array) < self._upload_session.total_parts:
+            chunk = self._read_chunk()
+            self._uploaded_part = self._upload_session.upload_part_bytes(chunk, len(self._part_array) * self._upload_session.part_size,
```

Comment:
This line looks weird in GitHub; could you check the formatting?  The list of arguments might need to be broken onto multiple lines